modules/registry-refactoring/src/main/java/org/apache/airavata/registry/core/entities/expcatalog/ComputeResourceScheduleEntity.java

```diff
@@ -17,31 +17,70 @@
  * specific language governing permissions and limitations
  * under the License.
  *
-*/
+*//*
+
 package org.apache.airavata.registry.core.entities.expcatalog;
 
 import javax.persistence.*;
+import java.io.Serializable;
+
+*/
+/**
+ * The persistent class for the compute_resource_schedule database table.
+ *//*
 
 @Entity
-@Table(name = "EXPCAT_COMPUTE_RESOURCE_SCHEDULING")
-public class ComputeResourceSchedulingEntity {
+@Table(name = "COMPUTE_RESOURCE_SCHEDULE")
```

Comment:
This table is called USER_CONFIGURATION_DATA, not COMPUTE_RESOURCE_SCHEDULE.  See https://github.com/apache/airavata/blob/develop/modules/registry/registry-core/src/main/resources/expcatalog-mysql.sql#L178.

I'm afraid that wherever these EXPCAT_* tables came from are very outdated. You should use expcatalog-mysql.sql from registry-core as the source of truth about what these tables should look like.